webapi/src/main/scala/org/knora/webapi/responders/v2/resources/CreateResourceV2Handler.scala

```diff
@@ -342,11 +312,13 @@ final case class CreateResourceV2Handler(
                                        cardinalityInfo.cardinality == ExactlyOne || cardinalityInfo.cardinality == AtLeastOne
                                      }.keySet -- resourceClassInfo.linkProperties
 
-      internalPropertyIris: Set[SmartIri] = internalCreateResource.values.keySet
+      propertiesWithoutValues: Set[SmartIri] = internalCreateResource.values.filter { case (_, values) =>
+                                                 values.nonEmpty
+                                               }.keySet
```

Comment:
This is the actual bugfix: Checking the presence of values rather than the presence of properties in the create request for mandatory properties.